src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/RiskService.kt

```diff
@@ -86,8 +87,10 @@ internal class RiskService(
   suspend fun getAssessmentStatus(crn: String): String {
     val assessmentsResponse = fetchAssessments(crn)
     val latestAssessment =
-      assessmentsResponse.assessments?.sortedBy { LocalDateTime.parse(it.initiationDate).toLocalDate() }?.reversed()
-        ?.firstOrNull()
+      assessmentsResponse.assessments
+        ?.map { it.copy(initiationDate = it.initiationDate ?: DEFAULT_DATE_TIME_FOR_NULL_VALUE) }
+        ?.sortedBy { it.initiationDate }
+        ?.lastOrNull()
```

Comment:
We don't need the map.  Just do:

?.sortedBy { it.initiationDate ?: DEFAULT_DATE_TIME_FOR_NULL_VALUE }

Reply:
Done